sapi/fpm/tests/tester.inc

```diff
@@ -589,7 +589,8 @@ class Tester
         string $successMessage = null,
         string $errorMessage = null,
         bool $connKeepAlive = false,
-        string $scriptFilename = null
+        string $scriptFilename = null,
+        $stdin = false
```

Comment:
The type is missing and the PHPDoc has not been updated.

Reply:
Right. I've looked into `request_data`, and it turns out I can use `null|string` rather than `false|string`.